spec/features/controllers/api/v1/reviews_controller_spec.rb

```diff
@@ -0,0 +1,73 @@
+require "rails_helper"
+
+RSpec.describe Api::V1::ReviewsController, type: :controller do
+
+    describe "POST#create" do
+        let!(:trail_1) { Trail.create(
+            name: "Mt Greylock", 
+            city: "Amherst", 
+            zip: "01234", 
+            route_type: "Loop", 
+            length: "3km",
+            elevation: "3km",
+            difficulty: "9",
+            description: "A description!"
+        ) }
+        let!(:user_1) { User.create(
+            email: "email1@email.com",
+            username: "username",
+            password: "password",
+            zip: "01234"
+        )}
+        let!(:review_1) { Review.create(
+            rating: "4",
+            body: "Great hike! Could have been tidier.",
+            trail: trail_1,
+            user: user_1
+        ) }
+
+        it "should return a successful status" do
+            get :create, params: {trail_id: trail_1.id, current_user: user_1, review: { rating: "4", body: "Great hike! Could be tidier."}}
```

Comment:
this should be a post instead of `get`, and we probably need to sign in the user as well